test/httpClientTestFixture.js

```diff
@@ -129,6 +129,25 @@ function setupBasicHttpClientTests({clientFunction, requestScoped = false}) {
       .then(() => tracer.expectNextSpanToEqual(testClient.successSpan({path})));
   });
 
+  it('should not report errors on 1xx', () => {
+    const path = '/weather/taipei';
+    return testClient.get({path})
+      .then(() => tracer.expectNextSpanToEqual({
+        name: 'get',
+        kind: 'CLIENT',
+        localEndpoint: {serviceName: localServiceName},
+        remoteEndpoint: {serviceName: remoteServiceName},
+        tags: {
+          'http.path': path,
+          'http.status_code': '101',
+        }
+      }))
+      .catch((err) => {
```

Comment:
odd you would need a catch here as the promise should bubble up. did you try the debug approach in the README? sometimes slowing down by using debugger can help faster vs guessing